Date:   Tue Oct 10 19:45:25 2023 -0400

    Disabled CMake's built-in package finder for select packages
    
    * CMake's find_package is not aware of Nuget dependencies and
      on Windows checks for packages in odd places, such as under
      _Program Files_. All such checks were disabled for packages
      that will be installed via Nuget - zLib and Expat.

diff --git a/cmake/findDependencies.cmake b/cmake/findDependencies.cmake
index b0897c323..82eeece27 100644
--- a/cmake/findDependencies.cmake
+++ b/cmake/findDependencies.cmake
@@ -45,7 +45,7 @@ if (APPLE)
 endif()
 
 if( EXIV2_ENABLE_PNG )
-    find_package( ZLIB REQUIRED )
+    #find_package( ZLIB REQUIRED )
 endif( )
 
 if( EXIV2_ENABLE_BMFF AND EXIV2_ENABLE_BROTLI )
@@ -62,7 +62,7 @@ if (EXIV2_ENABLE_XMP AND EXIV2_ENABLE_EXTERNAL_XMP)
     message(FATAL_ERROR "EXIV2_ENABLE_XMP AND EXIV2_ENABLE_EXTERNAL_XMP are mutually exclusive.  You can only choose one of them")
 else()
     if (EXIV2_ENABLE_XMP)
-        find_package(EXPAT REQUIRED)
+        #find_package(EXPAT REQUIRED)
     elseif (EXIV2_ENABLE_EXTERNAL_XMP)
         find_package(XmpSdk REQUIRED)
     endif ()
diff --git a/cmake/generateConfigFile.cmake b/cmake/generateConfigFile.cmake
index 3dd2fc934..6c49b6868 100644
--- a/cmake/generateConfigFile.cmake
+++ b/cmake/generateConfigFile.cmake
@@ -21,7 +21,7 @@ else()
     set(EXV_HAVE_XMP_TOOLKIT OFF)
 endif()
 set(EXV_HAVE_ICONV       ${ICONV_FOUND})
-set(EXV_HAVE_LIBZ        ${ZLIB_FOUND})
+set(EXV_HAVE_LIBZ        TRUE)
 set(EXV_HAVE_BROTLI      ${BROTLI_FOUND})
 
 if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.30.0")
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 8fe7366c6..cdfa7ba83 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -176,15 +176,15 @@ set_target_properties( exiv2lib_int PROPERTIES
 )
 
 # NOTE: Cannot use target_link_libraries on OBJECT libraries with old versions of CMake
-target_include_directories(exiv2lib_int PRIVATE ${ZLIB_INCLUDE_DIR})
+#target_include_directories(exiv2lib_int PRIVATE ${ZLIB_INCLUDE_DIR})
 target_include_directories(exiv2lib SYSTEM PRIVATE
     $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/xmpsdk/include>
 )
 
 if (EXIV2_ENABLE_XMP OR EXIV2_ENABLE_EXTERNAL_XMP)
-    target_include_directories(exiv2lib PRIVATE ${EXPAT_INCLUDE_DIR})
-    target_link_libraries(exiv2lib PRIVATE EXPAT::EXPAT)
-    list(APPEND requires_private_list "expat")
+    #target_include_directories(exiv2lib PRIVATE ${EXPAT_INCLUDE_DIR})
+    #target_link_libraries(exiv2lib PRIVATE EXPAT::EXPAT)
+    #list(APPEND requires_private_list "expat")
 endif()
 
 if (EXIV2_ENABLE_XMP)
@@ -233,8 +233,8 @@ else()
 endif()
 
 if( EXIV2_ENABLE_PNG )
-    target_link_libraries( exiv2lib PRIVATE ZLIB::ZLIB)
-    list(APPEND requires_private_list "zlib")
+    #target_link_libraries( exiv2lib PRIVATE ZLIB::ZLIB)
+    #list(APPEND requires_private_list "zlib")
 endif()
 
 if( EXIV2_ENABLE_BMFF AND BROTLI_FOUND )
